qiita_pet/templates/artifact_ajax/artifact_summary.html

```diff
@@ -104,7 +107,11 @@
       <i id='summary-title'>{{name}}</i><i> (ID: {{artifact_id}}) Visibility: {{visibility}}</i>
       {% if editable %}
         <a class="btn btn-default btn-sm" data-toggle="modal" data-target="#update-artifact-name"><span class="glyphicon glyphicon-pencil"></span> Edit</a>
-        <a class="btn btn-default btn-sm" onclick="load_process_artifact_ui({{artifact_id}});"><span class="glyphicon glyphicon-play"></span> Process</a>
+        {% if artifact_type == 'BIOM' %}
+          <a class="btn btn-default btn-sm" onclick="send_samples_to_analysis(this, [{{artifact_id}}]);"> Add to analysis</a>
```

Comment:
Will this display any indicator that the samples have been added to a study, like in the study list page?

Reply:
it uses the same code so it looks exactly the same!

Reply:
👍